packages/-ember-data/tests/integration/relationships/has-many-test.js

```diff
@@ -848,7 +848,7 @@ module('integration/relationships/has_many - Has-Many Relationships', function (
     });
   });
 
-  test('A hasMany relationship can be reloaded if it was fetched via a link', function (assert) {
+  test('A hasMany relationship can be reloaded if it was fetched via a link', async function (assert) {
```

Comment:
Sorry for the noise, this one was just a refactor, getting rid of run().

Reply:
No apology necessary, thanks a bunch for the cleanup?